common/src/stack/command/stack/commands/load/plugin_vm.py

```diff
@@ -0,0 +1,88 @@
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+import stack.commands
+from pathlib import Path
+
+class Plugin(stack.commands.Plugin):
+
+	def provides(self):
+		return 'vm'
+
+	def requires(self):
+		return [ 'host' ]
+
+	def run(self, section):
+
+		# Add virtual machine hosts
+		for vm in section:
+			host  = vm.get('name')
+			params = {
+				'hypervisor' : vm.get('hypervisor'),
+				'memory'     : vm.get('memory'),
+				'cpu'        : vm.get('cpu')
+			}
+
+			self.owner.stack('add.vm', host, **params)
```

Comment:
your `vm.get` calls will return None if the attr doesn't exist in the data. Does `self.owner.stack` handle that correctly, if it is passed a parameter like `memory=None`? Or should it be getting a blank string by default, IE: `memory=''`?

Reply:
The `add vm` command doesn't allow a VM to be added without those attributes so I was thinking that shouldn't be a possibility. However I probably should be handling that in case a dump is missing it for whatever reason.

Reply:
Alright fixed this as well as removing `disk_names` list comprehension when I moved over to using a new parameter of `add vm storage` now.